### PR TITLE
Old VM tests used in EVM JIT development

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: test/vmIOandFlowOperationsTestFiller.json [hunk 1]
```diff
@@ -3003,5 +3003,230 @@
...
+            "origin" : "cd1722f3947def4cf144679da39c4c32bdc35681",
+            "caller" : "cd1722f3947def4cf144679da39c4c32bdc35681",
+            "value" : "1000000000000000000",
+            "data" : "",
+            "gasPrice" : "100000000000000",
+            "gas" : "10000"
+        }
+    },
+
+    "byte1": {
```
Comment (CJentzsch) [inline test/vmIOandFlowOperationsTestFiller.json]: the same name for two tests wont work.

**Comment by CJentzsch** [comment]
other than that, looks good.

File: test/vmIOandFlowOperationsTestFiller.json (hunk 1 above)
Comment (chfast) [inline test/vmIOandFlowOperationsTestFiller.json]: I missed that

**Comment by chfast** [comment]
Added more tests, that should be all I have.

**Pull request merged**